targetconfig.json

```
@@ -478,6 +478,12 @@
                     "devUrl": "http://localhost:5173",
                     "aspectRatio": 3.45
                 }
+            },
+            "microbit-apps/pxt-arcadeshield": {
+                "simx": {
+                    "sha": "4055519e34baee00656d83b963dc1ff3a104afa2",
```

I believe this is the sha you want, the most recent publish to ghpages:
```suggestion
                    "sha": "98c47e4eaeceeb6bf0e26099743df2fdd2b5dd50",
```

---

The steps are outlined here: https://github.com/microsoft/pxt-simx-sample/tree/master/simx#releasing-your-simulator-extension. I probably should move this doc to pxt repo for discoverability.

---

I am confused about which commit hash to use. It should be the one associated to the gh-pages deploy?

---

how do I coordinate release of extension? Separate issue?

---

Changes to targetconfig.json go live upon PR merge, or pretty soon thereafter.

---

Please let me know if the doc linked above doesn't clearly indicated which hash to use. I've tried to ensure it communicates this as clearly as possible. If it isn't there yet, I'd like to address it.